src/lib/isRgbColor.js

```diff
@@ -1,12 +1,40 @@
+/* eslint-disable prefer-rest-params */
 import assertString from './util/assertString';
 
 const rgbColor = /^rgb\((([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5]),){2}([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\)$/;
 const rgbaColor = /^rgba\((([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5]),){3}(0?\.\d|1(\.0)?|0(\.0)?)\)$/;
 const rgbColorPercent = /^rgb\((([0-9]%|[1-9][0-9]%|100%),){2}([0-9]%|[1-9][0-9]%|100%)\)/;
 const rgbaColorPercent = /^rgba\((([0-9]%|[1-9][0-9]%|100%),){3}(0?\.\d|1(\.0)?|0(\.0)?)\)/;
+const startsWithRgb = /^rgba?/;
 
-export default function isRgbColor(str, includePercentValues = true) {
+export default function isRgbColor(str, options) {
   assertString(str);
+  // remove spaces
+  let strict, includePercentValues;
+  if (typeof options === 'object') {
+    strict = options.strict !== undefined ? options.strict : true;
+    includePercentValues = options.includePercentValues !== undefined ?
+      options.includePercentValues : true;
+  } else {
+    // backward compaitable behaviour
+    // Defaults
```

Comment:
I think we can make the object the default behaviour, as long as the backwards compatible behaviour keeps working this would not be a breaking change. But I'm interested what @braaar and @rubiin have to say about this

Reply:
@a-h-i I still think it's worthwhile to flip the logic here and put the object in the `else` clause

Reply:
@braaar changed in [9d0d3b3](https://github.com/validatorjs/validator.js/pull/2029/commits/9d0d3b3d6c720af8b0725cd7e0e3c7c2cb50b859)